presto-hive/src/main/java/com/facebook/presto/hive/metastore/glue/GlueHiveMetastore.java

```diff
@@ -205,12 +207,23 @@ else if (config.getPinGlueClientToCurrentRegion()) {
 
     @Override
     public Optional<Table> getTable(String databaseName, String tableName)
+    {
+        return getGlueTable(databaseName, tableName).map(table -> GlueToPrestoConverter.convertTable(table, databaseName));
+    }
+
+    private com.amazonaws.services.glue.model.Table getGlueTableOrElseThrow(String databaseName, String tableName)
```

Comment:
why the fully specified path?

Reply:
`com.facebook.presto.hive.metastore.Table` is already importet as `Table` in this class